### DISCUSSION TITLE
GitHub Discussions を利用するとなった場合

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#13] [要望・アイデア・不具合]
本運用するとなった場合、GitHub 上にも利用規約（などの定型約款）の記載があったほうがいいと思います。
https://qiita.com/terms ←こういうやつ！

**Comment by kskwtnk** [comment]
@blono 

ありがとうございます！
こちらについては間違いありませんね、本運用の際にはこういった点もしっかり準備いたします！
